docs/static/monitoring/monitoring-mb.asciidoc

```diff
@@ -23,28 +25,28 @@ Want to use {agent} instead? Refer to <<monitoring-with-elastic-agent>>.
 ==== Disable default collection of {ls} monitoring metrics
 
 --
-The `monitoring` setting is in the {ls} configuration file (logstash.yml), but is
-commented out: 
+Set the `monitoring.enabled` to `false` in logstash.yml to disable to default monitoring:
 
 [source,yaml]
 ----------------------------------
 monitoring.enabled: false
 ----------------------------------
 
-Remove the `#` at the beginning of the line to enable the setting.
-
 --
 
 [float]
 [[define-cluster__uuid]]
-==== Define `cluster_uuid` (optional)
-To bind the metrics of {ls} to a specific cluster, optionally define the `monitoring.cluster_uuid`
+==== Define `cluster_uuid`
+If your each pipeline contains <<plugins-outputs-elasticsearch,{es} output plugin>>,
+*Stack Monitoring* automatically figures out the `cluster_uuid` and no need to specify it.
+To bind the metrics of {ls} to a specific cluster, define the `monitoring.cluster_uuid`
```

Comment:
```suggestion
You will need to determine which Elasticsearch cluster that {ls} will bind metrics to in the Stack Monitoring UI by specifying the `cluster_uuid`.
When pipelines contain <<plugins-outputs-elasticsearch,{es} output plugins>>, the `cluster_uuid` is automatically calculated, and the metrics should be bound without any additional settings.

To override automatic values, or if your pipeline does not contain any <<plugins-outputs-elasticsearch,{es} output plugins>>, you can bind the metrics of {ls} to a specific cluster, by defining the target cluster in the `monitoring.cluster_uuid` setting.
```

Reply:
Awesome, thank you for this, much much better.